Sources/AppStoreConnectCLI/Services/Operations/InviteTesterOperation.swift

```diff
@@ -26,7 +26,7 @@ struct InviteTesterOperation: APIOperation {
         self.options = options
     }
 
-    func execute(with requestor: EndpointRequestor) throws -> AnyPublisher<BetaTester, Error> {
+    func execute(with requestor: EndpointRequestor) throws -> AnyPublisher<GetBetaTesterOperation.Output, Error> {
```

Comment:
This should probably just be returning the beta tester SDK model and then we can call the get operation in the service. That way we get that flat operation model and this operation is not referring to the output of another operation. What do you think?